doc/nrf/gs_ins_windows.rst

```diff
@@ -273,8 +316,41 @@ To install those, open a |bash| in the ``ncs`` folder and enter the following co
 Setting up the build environment
 ********************************
 
+Before you start :ref:`building and programming a sample application <gs_programming>`, you must set up your build environment.
+
+Setting up the SES environment
+==============================
+
+If you plan to :ref:`build with SEGGER Embedded Studio <gs_programming>`, the first time you import an |NCS| project, SES will prompt you to set the paths to the Zephyr Base directory and the GNU ARM Embedded Toolchain.
+This must be done only once per project.
+
+Complete the following steps to set up the |SES| environment:
+
+1. Run the file :file:`bin/emStudio`.
+
+#. Select **File -> Open nRF Connect SDK Project**.
+
+    .. figure:: images/ses_open.png
+       :alt: Open nRF Connect SDK Project menu
+
+       Open nRF Connect SDK Project menu
+
+#. Set the Zephyr Base directory to the full path to ``ncs\zephyr``.
+   The GNU ARM Embedded Toolchain directory is the directory where you installed the toolchain (for example, ``c:\gnuarmemb``).
+
+    .. figure:: images/ses_notset.png
+       :alt: Zephyr Base Not Set prompt
+
+       Zephyr Base Not Set prompt
+
+
+If you want to change these settings later, click **Tools -> Options** and select the **nRF Connect** tab (see :ref:`ses_options_figure`).
+
```

Comment:
A bit out of context, but if you have set up SES to test, could you retake that screenshot that is linked here with the correct GNU ARM Embedded Toolchain Directory? I had a typo in my path. ;-)

Reply:
Will do this when working on Troubleshooting SES section.